internal/user/repo_pg.go

```diff
@@ -0,0 +1,80 @@
+package user
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	repomodel "github.com/porky256/mock-interview-tbr/internal/models/repo"
+)
+
```

Comment:
I left this name "user_repo", because it will be easier to search for this file and work in IDE with several repo files for example

Reply:
in folder user will be file named user_...? seems a bit odd to me, every IDE will show you some postfix path if you will open same named files

Reply:
as you wish, I thought it would be easier

Reply:
i've not tried but probably even linter will shout on it

Reply:
resolved